src_assets/common/assets/web/PlatformLayout.vue

```diff
@@ -0,0 +1,27 @@
+<script setup>
+const props = defineProps({
+  platform: {
+    type: String,
+    required: true
+  }
+})
+
+</script>
+<template>
+<template v-if="$slots.windows && platform === 'windows'">
+  <slot name="windows"></slot>
+</template>
+
+<template v-if="$slots.linux && platform === 'linux'">
+  <slot name="linux"></slot>
+</template>
+
+<template v-if="$slots.macos && platform === 'macos'">
+  <slot name="macos"></slot>
+</template>
+</template>
+
+
+<style scoped>
+
+</style>
```

Comment:
```suggestion
</style>

```

Please ensure there is a blank line at the end of each file. You should be able to configure your IDE to handle this automatically.